src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier;
+
+    public Lottery() {
+        this.colorSupplier = new ColorSupplier();
```

Comment:
Field assignments within constructor should be based on arguments passed to constructor. In our case, we don't even want to assign it inside the constructor, instead use `private static final` keywords to define a constant.